spec/support/makefile_helper.rb

```diff
@@ -0,0 +1,7 @@
+module MakefileHelper
+  def self.dependencies(service_name)
+    filename = File.join("services", service_name, "Makefile")
+    rule = File.readlines(filename).first
+    rule.split(":")[1].scan(/[\w\-_]+/)
+  end
```

Comment:
Could we have a test for this helper? It's complicated enough that I think it would be worth it. Wouldn't bother for the ServicesHelper at this stage though.

Reply:
Hmm...I think it would be a bit 'meta' to test the test helpers 😆. I've modified it a bit to aid understanding - does that help?